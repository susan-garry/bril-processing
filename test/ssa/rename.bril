@main {
  v1: int = const 4;
  v2: int = const 0;
  mul1: int = mul v1 v2;
  add1: int = add v1 v2;
  v2: int = const 3;
  v3: int = const 0;
  print mul1;
  print add1;
  print v3;
}