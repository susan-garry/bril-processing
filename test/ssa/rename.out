yes
0
4
0
